app.js

```diff
@@ -0,0 +1,36 @@
+// Defines an express app that runs the boilerplate codebase.
+
+import express from 'express';
+import fetch from 'node-fetch';
+import path from 'path';
```

Comment:
interesting - the version we're using on the dashboards expects double quotes
https://github.com/Clever/schools-dashboard/blob/8e2d5e13eacb808a21492e4a44a957bd9d39a10b/.eslintrc.yml#L35